src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -90,6 +92,14 @@ function ref(string $id): ReferenceConfigurator
     return new ReferenceConfigurator($id);
 }
 
+/**
+ * Creates a parameter.
+ */
+function param(string $id): Parameter
+{
+    return new Parameter($id);
```

Comment:
should this return `'%'.$id.'%'`, so that one can do `'abc'.param($id).'def'`?
the `Parameter::__toString()` doesn't add the `%`

Reply:
The `$id` in parameter is without `%`.

Reply:
that's why I'm proposing to not use Parameter but
`return '%'.$id.'%';`

Reply:
Done